mod-invoice-storage/examples/document_collection.sample

```diff
@@ -0,0 +1,24 @@
+{
+  "documents": [
+    {
+      "id": "10a34f8a-98d1-45af-a9f6-14b7174ceb51",
+      "name": "sample.pdf",
+      "invoiceId": "733cafd3-895f-4e33-87b7-bf40dc3c8069",
+      "url": "http://www.africau.edu/images/default/sample.pdf",
```

Comment:
I don't think this should have a URL as the sample.pdf document is an attachment not a link.